projects/ama-sdk/src/lib/helpers/utils/logs-initiators.ts

```diff
@@ -0,0 +1,29 @@
+ /*!
+ * @license
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { InjectionToken } from '@angular/core';
+
+export const LOG_FILTER_ITEM_TOKEN = new InjectionToken<string>('log-filter-item');
+
+
+export function provideLogFilterItems(items: string[]) {
+    return {
+        provide: LOG_FILTER_ITEM_TOKEN,
+        multi: true,
+        useValue: items
+    };
+}
```

Comment:
We may create an "all" type LogMessageInitiator, what we can define in this file as well

```typescript
export const ALL_LOGS = Symbol('*');
export const allLogFilter: LogMessageInitiator  = {
        key: ALL_LOGS,
        displayName: 'SDK.ALL' // <-translatable, btw we should do the same for the others, please make them translatable
};
```